### ISSUE TITLE
whitespace inside templates causes logical failure

### ISSUE THREAD

**Issue by rgwozdz** [Esri/adlib#30]
Consider a template like:
```
{
  "title": "{{crashLayer.title || test}}"
}
```
And settings like:
```
{
  "crashLayer": {
    "title": "2008 Collisions"
  }
}
```

We get the result from adlib:
```
{
  "title": " test"
}
```
when what we really want is `"title": "2008 Collisions"`.  This occurs because leading/trailing whitespace in handlebars content is not being fully handled/trimmed during parsing.

**Comment by tomwayson** [comment]
This was closed via #31 right?

**Comment by jgravois** [comment]
released as [`v2.3.1`](https://github.com/Esri/adlib/releases/tag/v2.3.1) :tada:
